### PR TITLE
auth: don't double check subresource

### PR DESCRIPTION
**Pull request by mikedanese**
we already check that in authorizeStatusUpdate

```release-note
NONE
```

### CODE REVIEW

File: plugin/pkg/auth/authorizer/node/node_authorizer.go [hunk 1]
```diff
@@ -94,9 +94,7 @@ func (r *NodeAuthorizer) Authorize(attrs authorizer.Attributes) (authorizer.Deci
 			return r.authorizeGet(nodeName, configMapVertexType, attrs)
 		case pvcResource:
 			if r.features.Enabled(features.ExpandPersistentVolumes) {
-				if attrs.GetSubresource() == "status" {
```
Comment (liggitt) [inline plugin/pkg/auth/authorizer/node/node_authorizer.go]: dropping this means a get will fail

Comment (mikedanese) [reply plugin/pkg/auth/authorizer/node/node_authorizer.go]: Ahh, I see now.

Comment (mikedanese) [reply plugin/pkg/auth/authorizer/node/node_authorizer.go]: Unit tests passed :(

Comment (liggitt) [reply plugin/pkg/auth/authorizer/node/node_authorizer.go]: ah, looks like only integration tests were added for this.

**Pull request closed without merging**